simphony_mayavi/sources/tests/test_cuds_source.py

```diff
@@ -227,15 +227,18 @@ def test_mesh_source_from_vtk_mesh(self):
         self.assertIs(source.data, vtk_container.data_set)
         self.assertIs(vtk_cuds, vtk_container)
 
-    def test_mesh_source_name(self):
-        # given
-        mesh = Mesh('my_mesh')
-
+    def test_mesh_source_name_and_set_data_attributes(self):
         # when
-        source = CUDSSource(cuds=mesh)
+        # all data attributes are turned off except for point_scalars
+        source = CUDSSource(cuds=self.container,
+                            point_scalars="TEMPERATURE", point_vectors="",
+                            cell_scalars="", cell_vectors="")
```

Comment:
I should add more CUBA to the self.container.data because otherwise "TEMPERATURE" is the only available point_scalars and the test can hardly fail.